codalab/model/bundle_model.py

```diff
@@ -457,8 +457,9 @@ def get_bundle_uuids(self, conditions, max_results):
             if conditions['worksheet_uuid']:
                 # Select things on the given worksheet
                 clause = and_(clause, self.make_clause(cl_worksheet_item.c.worksheet_uuid, conditions['worksheet_uuid']))
-                clause = and_(clause, cl_worksheet_item.c.bundle_uuid == cl_bundle_metadata.c.bundle_uuid)  # Join
-                query = select([cl_bundle_metadata.c.bundle_uuid, cl_worksheet_item.c.id]).distinct().where(clause)
+                clause = and_(clause, cl_worksheet_item.c.bundle_uuid != None)
```

Comment:
Maybe add a comment saying that you're returning bundle uuids which aren't necessarily valid?